cmd/state/internal/cmdtree/projects.go

```diff
@@ -20,7 +20,10 @@ func newProjectsCommand(prime *primer.Values) *captain.Command {
 		[]*captain.Flag{},
 		[]*captain.Argument{},
 		func(ccmd *captain.Command, args []string) error {
-			return runner.Run()
+			if fail := runner.Run(); fail != nil {
+				return locale.NewError("tmp_err_runner_projects", fail.Error())
+			}
+			return nil
```

Comment:
You shouldn't need this. The error handling in the main package handles both types. If the middleware is depending on this that should be handled in the middleware because there'll likely be a few more runners that are returning failures.

Reply:
The issue with error chain handling that affected localized errors, also affects failures. The difference being that dealing with failures will be large enough of an issue, I highly suggest we go with this workaround and file a bug for smoothing this out in the short-term. (Projects and Push are the only two runners that return failures.)

I have two ideas (for the separate story):
1. Since failures can be shown to the user, failures should implement locale.ErrorLocalizer and possibly have an Unwrap method.
2. The main package handling of failures needs to reworked a bit to be able to unlink the first failure from the chain (though this would complicate the control flow of the current unwrapError function regarding existing localized error wrapping).

The above are just my current thoughts. I'm not sure how "in opposition" the above are, or if they may be needed in some combination.